roles/monitoring/tasks/main.yaml

```diff
@@ -50,15 +50,40 @@
   tags:
     - cluster-logging
 
+- name: V4M - check if storage class is being used
+  ansible.builtin.shell: |
```

Comment:
Just a note, used shell here rather than the ansible `kubernetes.core.k8s_info module` since PVs are not filterable using field selectors. This was the cleanest way to search if any PVs using a particular SC that I found.

Field Selector Support: https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/#list-of-supported-fields